.github/workflows/create_release.yml

```
@@ -0,0 +1,40 @@
+name: Create Release
+
+on:
+  push:
+    tags: ['v*']
+    branches: [master]
+  release:
+    types: [published]
```

We could also probably add a `pull_request` trigger here that just triggers if the workflow file itself is modified, just for testing purposes
```suggestion
    types: [published]
  pull_request:
    paths:
      - .github/workflows/create_release.yml
```

---

Done, but needed adjustment in the tag name extraction